server/mail.js

```diff
@@ -13,7 +13,7 @@ const transporter = nodemailer.createTransport({
 
 const sendEmail = async ({ recipient, subject, message }) => {
 	return await transporter.sendMail({
-		from: "no_reply@example.com",
+		from: "no_reply@love-me-tender.com",
```

Comment:
Did you test that sending the e-mail works?

Reply:
the test model works , but not in prod version , I couldn't find a free smtp provider .

Reply:
What do you mean by test model?

Reply:
When I test sending an email with Mailtrap, I see the email in the inbox, but I cannot send an email to a real person because Mailtrap does not provide this service. I tried to use Mailgun and SendGrid, but they didn't have the service either.

Reply:
Did you try this? https://edigleyssonsilva.medium.com/how-to-send-emails-securely-using-gmail-and-nodejs-eef757525324 

I added a gmail address to our Bitwarden.